landcover.mss

```diff
@@ -278,7 +278,8 @@
 
   [feature = 'landuse_farm'],
   [feature = 'landuse_farmland'],
-  [feature = 'landuse_greenhouse_horticulture'] {
+  [feature = 'landuse_greenhouse_horticulture'],
+  [feature = 'plant_nursery'] {
```

Comment:
@nebulon42 Fine, I edit it with notepad++ and use "github desktop" and this is a mistake... I revert and redo change... Sorry. So Kosmtik is a good idea to start with style. Have you a tuto to start with it?

Thanks